### ISSUE TITLE
Request for upload/dropzone

### ISSUE THREAD

**Issue by visualcookie** [vuikit/vuikit#100]
Upload/Dropzone is not implemented yet in vuikit, but it can already be found in UIkit. Is it possible to add it?

Doc article of UIkit: https://getuikit.com/docs/upload

**Comment by miljan-aleksic** [comment]
I believe that UIkit component is out of scope, a UI library should not enforce nor focus on any upload system, thus, it will not be ported to Vuikit. But I'm not discarding the drop area part.

**Comment by visualcookie** [comment]
Will the dropzone work like a dropzone? E.g. it has a prop for `upload-url` which points to the API url where the upload stuff happens? I've seen that in alot of Vuejs based upload fields, but none matches the style of UIkit sadly.

**Comment by miljan-aleksic** [comment]
Yeah, something like that. I will work on it as soon as I get the chance. It should be quite easy to create your own version, though, if you need help on it let me know.
